src/UrlStore.php

```diff
@@ -4,13 +4,14 @@
 
 namespace Horde\Core;
 
-use Horde;
 use Horde_Url;
+use Horde;
+Use Horde_Registry;
 
-class UrlStore
+class UrlStore 
```

Comment:
This class needs Horde_Registry as a dependency in the constructor. I think I also forgot that in my version.
See the RedirectToLogin Middleware constructor for reference.